morphir/runtime/src/org/finos/morphir/runtime/MorphirRuntime.scala

```diff
@@ -9,34 +9,33 @@ import org.finos.morphir.runtime.environment.MorphirEnv
 import org.finos.morphir.runtime.exports.RTAction
 import org.finos.morphir.runtime.quick.QuickMorphirRuntime
 import org.finos.morphir.runtime.MorphirRuntimeError.*
+import org.finos.morphir.ir.Value.TypedValue
 
 trait MorphirRuntime {
-  type TypeAttribs
```

Comment:
These still look wrong to me - the thing is, these never referred to `TypedMorphirRuntimeDefs` at all. I don't think anything in this file needed to change.

Reply:
Previously I attempted to keep these here, but in the file `TypedMorphirRuntimes`, it implements these signatures. It is expecting definitions for TypeAttribs + ValueAttribs, and it was pulling those definitions from the erradicated `TypedMorphirRuntimeDefs` . Since this file has been removed, there is no longer an implementation for TypeAttribs + ValueAttribs.

Reply:
Okay, I see. You still don't need to remove these definitions or change this file, but rather than removing the associated bindings in `TypedMorphirRuntimes`, you can probably just bind them to `Unit` and `UType`. (I added a comment on the relevant lines).